sm2/Cargo.toml

```
@@ -33,13 +33,14 @@ proptest = "1"
 rand_core = { version = "0.6", features = ["getrandom"] }
 
 [features]
-default = ["arithmetic", "dsa", "pem", "std"]
+default = ["arithmetic", "dsa", "pke", "pem", "std"]
 alloc = ["elliptic-curve/alloc"]
 std = ["alloc", "elliptic-curve/std", "signature?/std"]
 
 arithmetic = ["dep:primeorder", "elliptic-curve/arithmetic"]
 bits = ["arithmetic", "elliptic-curve/bits"]
 dsa = ["arithmetic", "dep:rfc6979", "dep:signature", "dep:sm3"]
+pke = ["arithmetic", "dep:rfc6979", "dep:sm3"]
```

The `dep:rfc6979` doesn't appear to be necessary here:

```suggestion
pke = ["arithmetic", "dep:sm3"]
```